lib/ex_unit/lib/ex_unit/runner_stats.ex

```diff
@@ -25,6 +25,13 @@ defmodule ExUnit.RunnerStats do
     {:noreply, %{map | total: total + 1, skipped: skipped + 1}}
   end
 
+  def handle_cast({:case_finished, %ExUnit.TestCase{name: case_name, state: {:failed, _reason}}}, %{failures: failures, total:
```

Comment:
Can you please move the second argument to its own line? something like:

    def handle_cast({:case_finished, %ExUnit.TestCase{name: case_name, state: {:failed, _reason}}}, 
                    %{failures: failures, ...})